packages/abi-gen/test-cli/expected-output/python/abi_gen_dummy.py

```diff
@@ -61,22 +94,48 @@ def simple_require(
             view_only=True
         )
 
+    def accepts_an_array_of_bytes(
+        self,
+        a: List[bytes],
+        tx_params: Optional[TxParams] = None,
+    ) -> None:
+        """Execute underlying, same-named contract method.
+
+        :param tx_params: transaction parameters
+
+        """
+        a = [
+            bytes.fromhex(a_element.decode("utf-8"))
+            for a_element in a
+        ]
+        func = self._get_contract_instance(
+            self.contract_address
+        ).functions.acceptsAnArrayOfBytes(
+            a
+        )
+        return self._invoke_function_call(
+            func=func,
+            tx_params=tx_params,
+            view_only=True
+        )
+
     def ecrecover_fn(
         self,
-        hash: bytes,
+        _hash: bytes,
```

Comment:
Why was the underscore added in this case?

Reply:
Because hash is a built-in function in Python, so it shouldn't be used as an identifier name, or else `pylint` will complain about it.  It was added by `abi-gen`'s `utils.toPythonIdentifier()`, which was added in the commit titled `
abi-gen: convert Py method params to snake case`.